From 9f9d999bed5429e09b62945706cd0b5f6d13ab2e Mon Sep 17 00:00:00 2001
From: Jack Ping CHNG <jack.ping.chng@linux.intel.com>
Date: Thu, 28 Jan 2021 11:56:42 +0800
Subject: [PATCH] DRV_DPM_SW-458-kernel errors on cat sys kernel debug dp qos

Changed log level for get_dp_port_info deq_port_num since
it is not a real error
---
 drivers/net/datapath/dpm/gswip31/datapath_ppv4_api.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/net/datapath/dpm/gswip31/datapath_ppv4_api.c b/drivers/net/datapath/dpm/gswip31/datapath_ppv4_api.c
index 9313bac9e970..8522a7c8284d 100644
--- a/drivers/net/datapath/dpm/gswip31/datapath_ppv4_api.c
+++ b/drivers/net/datapath/dpm/gswip31/datapath_ppv4_api.c
@@ -2250,8 +2250,9 @@ int _dp_deq_port_res_get(struct dp_dequeue_res *res, int flag)
 
 	p_info = get_dp_port_info(res->inst, res->dp_port);
 	if (!p_info->deq_port_num) {
-		pr_err("%s get_dp_port_info %d fail\n",
-		       __func__, res->dp_port);
+		DP_DEBUG(DP_DBG_FLAG_QOS_DETAIL,
+			 "%s get_dp_port_info %d fail\n",
+			 __func__, res->dp_port);
 		return DP_FAILURE;
 	}
 
